### PR TITLE
feat: twikoo js slim

### PR DESCRIPTION
**Pull request by eallion**
* cloudbase use: twikoo.all.min.js
* other backend use: twikoo.min.js

* without cloudbase sdk, the network resource reduces 180kB for now

### CODE REVIEW

File: layouts/partials/comment.html [hunk 1]
```diff
@@ -211,8 +211,13 @@
...
             {{- $options = dict "minify" true | merge $options -}}
-            {{- $source := $cdn.twikooJS | default "lib/twikoo/twikoo.all.min.js" -}}
-            {{- dict "Source" $source "Fingerprint" $fingerprint "Defer" true | dict "Scratch" .Scratch "Data" | partial "scratch/script.html" -}}
+            {{- if eq $twikoo.cloudbase true -}}
+                {{- $source := $cdn.twikooAllJS | default "lib/twikoo/twikoo.all.min.js" -}}
+                {{- dict "Source" $source "Fingerprint" $fingerprint "Defer" true | dict "Scratch" .Scratch "Data" | partial "scratch/script.html" -}}
+                {{- else -}}
+                {{- $source := $cdn.twikooJS | default "lib/twikoo/twikoo.min.js" -}}
+                {{- dict "Source" $source "Fingerprint" $fingerprint "Defer" true | dict "Scratch" .Scratch "Data" | partial "scratch/script.html" -}}
+            {{- end -}}
```
Comment (HEIGE-PCloud) [inline layouts/partials/comment.html]: I am slightly worried about backwards compatibility here. People who are currently using cloudbase will have to add the config on upgrading the theme. Is it possible that we make the default package `twikoo.all.min.js` and people can opt in to use the smaller one if they are not using cloudbase?

**Review by HEIGE-PCloud — COMMENTED** [review]
Thanks for your pull request! Before this can get merged, we should probably also update the documentation to reflect this change under `exampleSite/content/posts/theme-documentation-basics/index.en.md` and `exampleSite/content/posts/theme-documentation-basics/index.zh-cn.md`

File: layouts/partials/comment.html (hunk 1 above)
Comment (eallion) [reply layouts/partials/comment.html]: Yes, we can set cloudbase to true in the config.

Comment (eallion) [reply layouts/partials/comment.html]: updated

**Pull request merged**